packages/notifications/src/useNotifications.ts

```diff
@@ -0,0 +1,36 @@
+import { createContext, createElement, PropsWithChildren, useContext } from 'react';
+
+export type NotificationType = 'error' | 'warn' | 'info' | 'success' | 'welcome';
+export interface NotificationOptions {
+  detail: string | unknown;
```

Comment:
Nit: SonarCloud complains about this
https://sonarcloud.io/project/issues?resolved=false&pullRequest=4741&id=DataBiosphere_terra-ui&open=AY579IL_8UV9StxhzyAO

`string | unknown` is effectively the same as `unknown`.

```suggestion
  detail: unknown;
```